usl_models/train_flood_model.ipynb

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
let's stick this someplace like a directory named 'examples' rather than at the package root